contracts/test/integration/utils/RoleTesting.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import {Core, Vcon, Volt, IERC20, IVolt} from "../../../core/Core.sol";
+import {MainnetAddresses} from "../fixtures/MainnetAddresses.sol";
+import {TribeRoles} from "../../../core/TribeRoles.sol";
+import {DSTest} from "./../../unit/utils/DSTest.sol";
+import {L2Core} from "../../../core/L2Core.sol";
+import {Core} from "../../../core/Core.sol";
+import {Vm} from "./../../unit/utils/Vm.sol";
+import {Strings} from "@openzeppelin/contracts/utils/Strings.sol";
+
+import {console} from "hardhat/console.sol";
```

Comment:
Forge-std exposes a console btw :)

Reply:
I would use forge, but whenever I use that new interface, it bricks on the hardhat compile because hardhat doesn't have the same remappings as forge. Once we deprecate hardhat, definitely will do!